src/check_executor.rs

```diff
@@ -100,13 +103,15 @@ async fn executor_loop(
     checker: Arc<impl Checker + 'static>,
     producer: Arc<impl ResultsProducer + 'static>,
     reciever: UnboundedReceiver<ScheduledCheck>,
+    region: String,
```

Comment:
Is it actually possible to make this a `&str` then we wouldn't need to clone it around quite as much I think